File: src/main/java/com/jason/student/Student.java
package com.jason.student;

public class Student {
    String id;   //
    String name; //
    int english; //
    int math;    //attribute 會給初值
    public Student(){
        super();
    }

    public Student(String name, int english, int math) {
        this.name = name;
        this.english = english;
        this.math = math;
    }
    public void print(){
        System.out.println(name + "\t" + english + "\t" + math + "\t" + (english+math)/2);
    }
}

File: src/main/kotlin/com/jason/student/Student.kt
package com.jason.student

import java.util.*

fun main(args: Array<String>) {

    val scanner = Scanner(System.`in`)   //在Kotlin裡 in 是關鍵字 ， 若JAVA屬性有用到Kotlin的關鍵字，在其左右加上`符號即可
    print("Please enter student's name : ")
    var name = scanner.next()
    print("Please enter student's english score: ")
    var english = scanner.nextInt()
    print("Please enter student's math score : ")
    var math = scanner.nextInt()
    val stu = Student(name,english,math)
    stu.print()
}

class Student(var name:String,var english:Int,var math:Int){
    fun print(){
        println(name + "\t" + english + "\t" + math + "\t" +(english+math)/2)
    }
}